config.rb

```diff
@@ -48,7 +48,7 @@
 # Build-specific configuration
 configure :build do
   # Minify CSS on build
-  activate :minify_css
+  # activate :minify_css
```

Comment:
There's a chance that it's related to https://github.com/alphagov/govuk-frontend/issues/1350

Reply:
Yeah its the same problem